### PR TITLE
Frontmatter Upgrade

### PR DESCRIPTION
**Pull request by ryanpeach**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@ debug = true
 aho-corasick = "1.1.3"
 bon = "2.3.0"
 clap = { version = "4.5.16", features = ["derive"] }
-comrak = "0.29.0"
+comrak = "0.31.0"
```
Comment (ryanpeach) [inline Cargo.toml:18]: Comrak needs a new release

Comment (kivikakk) [reply Cargo.toml:18]: [Done!](https://crates.io/crates/comrak/0.32.0)

Comment (ryanpeach) [reply Cargo.toml:18]: @kivikakk thanks!

**Comment by ryanpeach** [comment]
```
---
ugds: xlvf, zptqgmbety
---

- lorem lhet qfee trbiuhd bz #dolor
-
```

"dolor" starts at offset 62 but comrak says 61

```
- [[Tvtinl]]
    - Axndco/Pbyudhm
        - Zqxzn Avdxyxofu'm
    - Vgrjsat
    - Foueh eypvsd?
    - Xotw wlhcgrryrj
    - Cbvn ocjrla
    - Siuh lnmcnlv
    - Jztvzm 21 pdyhwjzvnx
- Hfxeno aq tmnz d lecc ymdeuz
    - Lh cy b brkbkgmtz frg ksocv gomu yh mvr'fw ysnt doeun bzim, eka aclmy vrpa hys oz tuk mpb nlle lyn icazyvey ik my peub xr
- [[Mek Xvmr]]
    - "Usp tcr itlo cb zsez hjmrou ler f pdv"
    - Pav bvqv egx ic jxa rto dd amr nwd. Hwt sq vabt kyzk hhx yp, tb j qyt cebfd:
    - Jjyyvwrro xkbj acor uzifhktovnah hfbyv, udh hluq hvv jx uoua'o whnp ik wxagb knyiyds
```

"icazyvey" begins at offset 318 but comrak says 220

@kivikakk @SamWilsn can you confirm RE: https://github.com/kivikakk/comrak/pull/494

**Comment by ryanpeach** [comment]
Relevant methods:

https://github.com/ryanpeach/mdlinker/blob/8dddb13f726a100ab7bb82a369ae69b26f932559/src/file/content/wikilink.rs#L95

https://github.com/ryanpeach/mdlinker/blob/8dddb13f726a100ab7bb82a369ae69b26f932559/src/rules/unlinked_text.rs#L164

**Comment by ryanpeach** [comment]
@kivikakk @SamWilsn RE: https://github.com/kivikakk/comrak/pull/494 Sorry, false alarm. Fixed my tests.

**Pull request merged**